doc/experiment.rst

```diff
@@ -1,327 +1,454 @@
+  To have SmartSim attempt to find a launcher on your machine, set the `launcher`
+  argument to `"auto"` during ``Experiment`` initialization like below:
 
-.. code-block:: bash
+  .. code-block:: python
 
-    #!/bin/bash
-    #SBATCH --exclusive
-    #SBATCH --nodes=1
-    #SBATCH --ntasks-per-node=32
-    #SBATCH --time=00:10:00
+      exp = Experiment("name-of-experiment", launcher="auto")  # auto-detect launcher
 
-    python /path/to/script.py
+  If a launcher cannot be found or no `launcher` parameter is provided, the default value of
+  `launcher="local"` will be used.
 
-Ensemble
 ========
+Entities
+========
+Entities are SmartSim API objects that can be launched and
+managed on the compute system via the Experiment API. While the
+``Experiment`` object is intended to be instantiated once in a
+SmartSim driver script, there is no limit to the number of SmartSim entities
+within an Experiment. In the following subsections, we define the
+general purpose of the three entities that can be created via
+Experiment API factory methods:
+  * ``Orchestrator``
+  * ``Model``
+  * ``Ensemble``
+
+To create a reference to each entity object, use the associated
+``Experiment.create_...()`` function.
+
+.. list-table:: Experiment API Entity Creation
+   :widths: 20 65 25
+   :header-rows: 1
+
+   * - Factory Method
+     - Example
+     - Return Type
+   * - ``create_database()``
+     - ``orch = exp.create_database([port, db_nodes, ...])``
+     - :ref:`Orchestrator <orchestrator_api>`
+   * - ``create_model()``
+     - ``model = exp.create_model(name, run_settings)``
+     - :ref:`Model <model_api>`
+   * - ``create_ensemble()``
+     - ``ensemble = exp.create_ensemble(name[, params, ...])``
+     - :ref:`Ensemble <ensemble_api>`
+
+Each entity instance can be used to start,
+monitor, and stop simulations from the notebook
+using the :ref:`Experiment API<experiment_api>`.
+
+.. list-table:: Interact with Entities during the Experiment
+   :widths: 25 55 25
+   :header-rows: 1
+
+   * - Experiment Function
+     - Example
+     - Desc
+   * - ``start()``
+     - ``exp.start(*args[, block, summary, ...])``
+     - Launch an Entity
+   * - ``stop()``
+     - ``exp.stop(*args)``
+     - Clobber an Entity
+   * - ``get_status()``
+     - ``exp.get_status(*args)``
+     - Retrieve Entity Status
+
+Orchestrator
+------------
+The orchestrator is an in-memory database with features designed
+to enable a wide variety of AI-enabled workflows, including features
+for online training, low-latency inference, cross-application data
+exchange, online interactive visualization, online data analysis, computational
+steering, and more. The ``Orchestrator`` can be thought of as a general
+feature store capable of storing numerical data, ML models, and scripts.
+The orchestrator is capable of performing inference and script evaluation on feature
+store data. Any SmartSim ``Model`` or ``Ensemble`` model can connect to the
```

Comment:
'Any SmarSim  model or Ensemble of models' ? Technically an ensemble is a group of models.

Reply:
I refer to a single ensemble model here since we can instruct a single ensemble model to connect to an orch, and then another does not need to!